stablehlo/tests/interpret_abs.mlir

```diff
@@ -1,36 +1,26 @@
-// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+// RUN: stablehlo-interpreter --interpret -split-input-file %s
 
-// CHECK-LABEL: Evaluated results of function: abs_op_test_si64
-func.func @abs_op_test_si64() -> tensor<3xi64> {
+func.func @abs_op_test_si64() {
   %operand = stablehlo.constant dense<[-2, 0, 2]> : tensor<3xi64>
   %result = stablehlo.abs %operand : tensor<3xi64>
-  func.return %result : tensor<3xi64>
-  // CHECK-NEXT: tensor<3xi64>
-  // CHECK-NEXT: 2 : i64
-  // CHECK-NEXT: 0 : i64
-  // CHECK-NEXT: 2 : i64
+  check.eq %result, dense<[2, 0, 2]> : tensor<3xi64>
+  func.return
```

Comment:
Just curious, is `func.return` required?

Reply:
I do not think we can have a block without terminator. 
Besides that, from POV of evaluation of a StableHLO program, `func.return` plays an important role in breaking the  interpreter loop [here](https://github.com/openxla/stablehlo/blob/7e9ce9af667a65e2755c1fffec2ddd201f92d98d/stablehlo/reference/Ops.cpp#L537)